xbmc/pvr/PVRContextMenus.cpp

```diff
@@ -266,8 +266,10 @@ namespace PVR
     {
       const CPVRRecordingPtr recording(item.GetPVRRecordingInfoTag());
       if (recording && !recording->IsDeleted() && !recording->IsInProgress())
-        return true;
-
+      {
+        CPVRGUIActions GUIActions;
+        return GUIActions.CanEditRecording(item);
```

Comment:
We use GUI actions as a singleton, so: `return CServiceBroker::GetPVRManager().GUIActions()->CanEditRecording(item);`